### ISSUE TITLE
Fatal error: Uncaught Error: Call to a member function toString() on null in FqcnMethodSniff.php:250

### ISSUE THREAD

**Issue by jeroennoten** [gskema/phpcs-type-sniff#10]
I tried this sniff by following the installation instructions. I only required the package and added `<rule ref="./vendor/gskema/phpcs-type-sniff/src/Sniffs/CompositeCodeElementSniff.php"/>` to `phpcs.xml`.

When I ran `phpcs`, I got the following error:

```
Fatal error: Uncaught Error: Call to a member function toString() on null in vendor/gskema/phpcs-type-sniff/src/Sniffs/CodeElement/FqcnMethodSniff.php:250
Stack trace:
#0 vendor/gskema/phpcs-type-sniff/src/Sniffs/CodeElement/FqcnMethodSniff.php(118): Gskema\TypeSniff\Sniffs\CodeElement\FqcnMethodSniff::reportNullableBasicGetter(Object(PHP_CodeSniffer\Files\LocalFile), Object(Gskema\TypeSniff\Inspection\Subject\ReturnTypeSubject), Object(Gskema\TypeSniff\Core\CodeElement\Element\ClassMethodElement), Object(Gskema\TypeSniff\Core\CodeElement\Element\ClassElement))
#1 vendor/gskema/phpcs-type-sniff/src/Sniffs/CodeElement/FqcnMethodSniff.php(77): Gskema\TypeSniff\Sniffs\CodeElement\FqcnMethodSniff->processMethod(Object(PHP_CodeSniffer\Files\LocalFile), Object(Gskema\TypeSniff\Core\CodeElement\Element\ClassMethodElement), Object(Gskema\TypeSniff\Core\CodeElement\Element\ClassElement))
#2 vendor/gskema/phpcs-type-sniff/src/Sniffs/CompositeCodeElementSniff.php(99): Gskema\TypeSniff\Sniffs\CodeE in vendor/gskema/phpcs-type-sniff/src/Sniffs/CodeElement/FqcnMethodSniff.php on line 250
```

Note: I'm using version 0.14.0.

**Comment by jeroennoten** [comment]
I was able to reproduce the error with this file:

```php
<?php

class Foo
{
    /**
     * @var string|null
     */
    private $foo;

    public function __construct(?string $foo)
    {
        $this->foo = $foo;
    }

    /**
     * Description.
     */
    public function getFoo(): ?string
    {
        return $this->foo;
    }
}
```

**Comment by gskema** [comment]
Should be fixed 0.14.1. Let me know if it works for you. If you intend to have doc blocks with description but no tags, you may want to use `FqcnMethodSniff.reportMissingTags` @jeroennoten
